src/EnergyPlus/WaterCoils.cc

```diff
@@ -1406,11 +1406,8 @@ namespace WaterCoils {
                     //        DesSatEnthAtWaterInTemp =PsyHFnTdbW(WaterCoil(CoilNum)%DesInletWaterTemp, &
                     //                                             PsyWFnTdpPb(WaterCoil(CoilNum)%DesInletWaterTemp,StdBaroPress))
 
-                    // Total Coil Load from Inlet and Outlet Air States.
+                    // Total Coil Load from Inlet and Outlet Air States (which include fan heat as appropriate).
                     WaterCoil(CoilNum).DesTotWaterCoilLoad = WaterCoil(CoilNum).DesAirMassFlowRate * (DesInletAirEnth - DesOutletAirEnth);
-                    if (CurSysNum > 0 && CurSysNum <= DataHVACGlobals::NumPrimaryAirSys) {
-                        WaterCoil(CoilNum).DesTotWaterCoilLoad = WaterCoil(CoilNum).DesTotWaterCoilLoad + PrimaryAirSystem(CurSysNum).FanDesCoolLoad;
-                    }
```

Comment:
Went over this several times. At this point coil sizing is complete and the coil inlet or outlet air temp already includes fan heat. So in the DesTotWaterCoilLoad = mDot * deltaH equation, the H already includes fan heat so this would double count.